modules/Core/src/main/java/org/terasology/rendering/nui/layers/ingame/inventory/InventoryGrid.java

```diff
@@ -38,17 +38,35 @@
 import java.util.List;
 
 /**
+ * A grid of {@link InventoryCell} used to display an inventory of an entity.
  */
 public class InventoryGrid extends CoreWidget {
-
+    /*
```

Comment:
I think you *can* actually write this as javadoc. Sure, it is private and a field, but maybe that would show up anyway? Try pressing `CTRL-Q` in IntelliJ when there is a usage of the `maxHorizontalCells` or another field inside this file. Does the javadoc pop up (after you go from `/*` to `/**` ? If so: handy!